### PR TITLE
Decommission the /firefox/unfck campaign page (Fixes #11891)

### LINKED ISSUE #11891: Decommission /firefox/retention/thank-you/ page
## Description

This page has had less than 2500 page views in the past 12 months. It looks like no one is sending traffic to it any longer.

https://www.mozilla.org/en-US/firefox/retention/thank-you/

Suggest redirecting to /firefox/.

---

#### Success Criteria
- [ ] Page is removed and redirected

### PR DESCRIPTION
**Pull request by alexgibson**
## One-line summary

Removes the `/firefox/unfck/` campaign page and redirects to `/firefox/`

## Issue / Bugzilla link

#11891

## Testing

- [ ] http://localhost:8000/en-US/firefox/unfck/ redirects to http://localhost:8000/en-US/firefox/
- [ ] No stray assets remaining

### CODE REVIEW

File: bedrock/firefox/redirects.py [hunk 1]
```diff
@@ -529,12 +529,12 @@ def firefox_channel(*args, **kwargs):
...
     redirect(r"^/firefox/windows-64-bit/?$", "firefox.browsers.windows-64-bit"),
     redirect(r"^/firefox/best-browser/?$", "firefox.browsers.best-browser"),
-    # Unfck campaign
-    redirect(r"^firefox/unfuck/?$", "firefox.campaign.unfck.index"),
-    redirect(r"^firefox/love/?$", "firefox.campaign.unfck.index"),
-    redirect(r"^firefox/liebe/?$", "firefox.campaign.unfck.index"),
-    redirect(r"^firefox/rendonslenetplusnet/?$", "firefox.campaign.unfck.index"),
-    redirect(r"^(unfu?ck|love|liebe|rendonslenetplusnet)/?$", "firefox.campaign.unfck.index"),
+    # Unfck campaign, issue 11613
+    redirect(r"^firefox/unfu?ck/?$", "firefox"),
```
Comment (reemhamz) [inline bedrock/firefox/redirects.py:533]: **Question:** why is there a `?` in the middle of `unfck`? Is it in the case if it was spelled with a `u` (making it the correct spelling 😝) to still redirect?

File: bedrock/foundation/templates/foundation/annualreport/2019/index.html [hunk 2]
```diff
@@ -542,8 +542,7 @@ <h2 class="mzp-c-card-feature-title"><span class="highlight green">Mozilla urged
...
             <p class="mzp-c-card-feature-desc">As part of returning Firefox to growth, we have to speak to people’s concerns about online life with our marketing and communications. 2020 revealed more than ever just how fed up internet users are with how their data is used by big platforms, and what little they get in exchange for giving up privacy to them.</p>
-            <p class="mzp-c-card-feature-desc">With this in mind we launched a <a href="{{ url('firefox.campaign.unfck.index') }}">campaign</a> with some easy choices for people to make to unfck their own corner of the internet. We believe those taking part will help drive meaningful change and create a healthier online life for all.</p>
-            <a target="_blank" rel="external noopener" class="mzp-c-button mzp-t-secondary" href="{{ url('firefox.campaign.unfck.index') }}">Learn more</a>
+            <p class="mzp-c-card-feature-desc">With this in mind we launched a campaign with some easy choices for people to make to unfck their own corner of the internet. We believe those taking part will help drive meaningful change and create a healthier online life for all.</p>
```
Comment (reemhamz) [inline bedrock/foundation/templates/foundation/annualreport/2019/index.html:545]: Should we mention here that it's been decommissioned?

**Review by reemhamz — APPROVED** [review]
Looks nice and decomissioned to me. Just a small suggestion, non-blocking.
Also, there are some links on some python files that mention the unfck Instagram page that we should probably also remove since the Instagram page has been deleted:
- https://github.com/mozilla/bedrock/blob/main/bedrock/settings/base.py#L972
- https://github.com/mozilla/bedrock/blob/main/bedrock/settings/base.py#L972
- https://github.com/mozilla/bedrock/blob/main/bedrock/settings/base.py#L972

r+wc 🦍

File: bedrock/firefox/redirects.py (hunk 1 above)
Comment (alexgibson) [reply bedrock/firefox/redirects.py:533]: It's a regular expression., so `?` means the preceding character is optional (meaning it matches both `/unfuck/` and `/unfck/`

File: bedrock/foundation/templates/foundation/annualreport/2019/index.html (hunk 2 above)
Comment (alexgibson) [reply bedrock/foundation/templates/foundation/annualreport/2019/index.html:545]: The campaign was more than just the landing page here, there were lots of different pieces of creative work that went into it. So the copy still applies well I think even without the link.

**Comment by alexgibson** [comment]
> Also, there are some links on some python files that mention the unfck Instagram page that we should probably also remove since the Instagram page has been deleted:

Good spot. It looks like these are still used in the footer, but the URL is currently 404 in prod: https://www.instagram.com/unfcktheinternet/

I'll file a follow up issue so we can figure out what this should be updated to

**Pull request merged**